regression-test/suites/load_p0/stream_load/test_json_load.groovy

```diff
@@ -794,7 +793,20 @@ suite("test_json_load", "p0") {
                 assertEquals("${reason}", "${out}")
             }
         }
+    } finally {
+        try_sql("DROP TABLE IF EXISTS ${testTable}")
+    }
+      
+    // iterate read json when read_json_by_line = false
+    try {
+        sql "DROP TABLE IF EXISTS ${testTable}"
 
```

Comment:
add cases like `iterate_read_json.json` but contains malformed json lines in the middle

Reply:
to make sure the simdjson exception is handled correctly